tests/OptionTypeTest.cpp

```diff
@@ -838,3 +838,16 @@ TEST_F(TApp, tupleTwoVectors) {
     EXPECT_EQ(std::get<0>(cv).size(), 2U);
     EXPECT_EQ(std::get<1>(cv).size(), 3U);
 }
+
+TEST_F(TApp, vectorSingleArg) {
```

Comment:
Shouldn't we have a test with a size 2+ vector as well?

Reply:
good call! on the additional test,  had to change the condition to make it work for that